### ISSUE TITLE
Sphinx breaking on notebook parsing

### ISSUE THREAD

**Issue by sbenthall** [econ-ark/HARK#412]
See https://github.com/econ-ark/HARK/issues/406#issuecomment-545008255

**Comment by sbenthall** [comment]
The breaking file is in the `HARK/cstwMPC` project directory, which is slated for removal. See PR #334 
It also appears to be a standalone script, rather than a library.
So it should not be involved in autogenerated API documentation--it is not specifying anything with an API.

I'll remove this and similarly misplaced files from the `autosummary` list.

**Comment by sbenthall** [comment]
Sphinx warning:
```
/home/sb/projects/econ-ark/HARK/Documentation/notebooks/DCT-Copula-Illustration.rst: WARNING: document not readable. Ignored.
```
The symlink here points to a file that's missing from the repository: `../../DCT-Copula-Illustration.ipynb`

That looks like an incorrect path anyway; there is a working symlink to `../../Examples/Gentle-Intro-To-HARK.ipynb`, but there is no `DCT-Copula-Illustration.ipynb` notebook in the `Examples` directory either.

**Comment by sbenthall** [comment]
Using `sphinx html`, `recommonmark` is erroring; it looks like it's erroring in the parsing of the HARKManual.
```
reading sources... [  4%] HARKmanual                                            
Exception occurred:
  File "/home/sb/anaconda3/lib/python3.6/site-packages/recommonmark/parser.py", line 211, in visit_image
    img_node['alt'] = ''.join(content)
TypeError: sequence item 1: expected str instance, NoneType found
```
There are also still `autosummary` errors.

As there seems to be a lot of work here I'll stop updating this issue and report the work in the upcoming PR.

**Comment by sbenthall** [comment]
The solution to the `recommonmark` error appears to be...removing the `recommonmark` extension.
Then sphinx executes without error and builds HTML files.

I'm not sure what `recommonmark` was doing and it may have been there for a good reason.

**Comment by sbenthall** [comment]
Fixed with #411
